Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -1893,6 +1912,185 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: 608bdaf3-5ea5-4ea7-85bc-0343f8535b0d
+    type: condition
+    task:
+      id: 608bdaf3-5ea5-4ea7-85bc-0343f8535b0d
+      version: -1
+      name: Is there a PPTX/XSLX file?
+      description: Checks if there is a Word file (DOC, DOCX) in context.
```

Comment:
The name and description do not match.
The name says it is looking for a **Powerpoint** or **Excel** file.
The description asks if a **Word** file was found.
Confusing.

Reply:
agree, missed that. TNX